test/e2e/common_suite_test.go

```diff
@@ -870,3 +874,253 @@ func doTestCreatePeerPodWithAuthenticatedImageWithoutCredentials(t *testing.T, a
 	pod := newPod(namespace, podName, podName, imageName, withRestartPolicy(v1.RestartPolicyNever))
 	newTestCase(t, "InvalidAuthImagePeerPod", assert, "Peer pod with Authenticated Image with Invalid Credentials has been created").withPod(pod).withAuthenticatedImage().withAuthImageStatus(expectedAuthStatus).withCustomPodState(v1.PodPending).run()
 }
+
+func doTestCaaDaemonsetUpgrade(t *testing.T) {
+	runtimeClassName := "kata-remote"
+	namespace := envconf.RandomName("default", 7)
+	deploymentName := "nginx-deployment"
+	containerName := "nginx"
+	imageName := "nginx"
```

Comment:
Shall we tag the testing tools and the nginx image and push them to https://quay.io/repository/confidential-containers/? So that we can avoid noisy docker hub limitations?